### PR TITLE
Added max button on stake & unstake field + show HALO amount to claim

### PR DESCRIPTION
**Pull request by schystz**
## Description of Changes
- Added max button on stake & unstake field on Farm page
- Show HALO amount to claim on Vesting page

UI tweaks:
- Updated copy on reward pink banner under stake/unstake field
- Updated bunny image on Harvest modal (bunny holding RNBW token)
- Implemented max button hover state

[Link to Jira Ticket](https://halodao.atlassian.net/browse/HDF-299)

### Developer Checklist:

* [x] I have followed the guidelines in our Contributing document
* [x] This PR has a corresponding JIRA ticket
* [x] My branch conforms with our naming convention i.e. `feature/HDF-XXX-description`
* [x] All files have appropriate file headers and documentation
* [ ] I have written new tests for your core changes, as applicable
* [x] I have successfully ran tests locally

### Reviewers Checklist:
* [ ] Code is readable and understandable; any unclear parts have explanations 
* [ ] UI/UX changes match the corresponding figma/other design resources, if applicable
* [ ] I have successfully ran tests locally

### CODE REVIEW

File: public/locales/en.json [hunk 1]
```diff
@@ -97,18 +97,21 @@
   "stakeable": "Stakable",
   "poolSummaryStakeable": "My Stakeable Value",
   "poolSummaryStaked": "My Staked Value",
-  "poolSummaryHaloEarned": "Unclaimed HALO Earned",
+  "poolSummaryHaloEarned": "Unclaimed RNBW Earned",
```
Comment (schystz) [inline public/locales/en.json:100]: @tracyarciaga any particular reason you changed this to RNBW? The unclaimed HALO we displayed on UI is converted to HALO actually (using the formula `unclaimedRNBW * currentHaloHaloPrice()`)

Comment (tracyarciaga) [reply public/locales/en.json:100]: ahh token names and symbol have been renamed - we're not using the name HALO anymore.

Comment (schystz) [reply public/locales/en.json:100]: Ok I see...do we wanna address that in this PR too? I think there are some more places we need to replace not just here.

Comment (tracyarciaga) [reply public/locales/en.json:100]: Yeah let's do it on another PR, sorry :) will approve this now

**Review by tracyarciaga — APPROVED** [review]
Approving test. I have tested it on my local and preview.

File: public/locales/en.json (hunk 1 above)
Comment (schystz) [reply public/locales/en.json:100]: No worries, just wanted to understand about this change to make sure there isn't any confusion. Thanks!

**Review by bitcoinbrisbane — CHANGES_REQUESTED** [review]
LGTM once the few unused vars are removed.

**Comment by schystz** [comment]
> LGTM once the few unused vars are removed.

Thanks for the catch @bitcoinbrisbane ! I didn't noticed it until you pointed it out. Fixed now!

**Review by bitcoinbrisbane — APPROVED** [review]

**Pull request merged**